packages/renderer/src/lib/help/HelpItems.ts

```diff
@@ -0,0 +1,109 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { faDiscord, faSlack } from '@fortawesome/free-brands-svg-icons';
+import { faComment } from '@fortawesome/free-regular-svg-icons';
+import {
+  faExclamationTriangle,
+  faExternalLinkAlt,
+  faLightbulb,
+  faListUl,
+  type IconDefinition,
+} from '@fortawesome/free-solid-svg-icons';
+
+export enum ActionKind {
+  LINK,
+  COMMAND,
+}
+
+export interface ItemAction {
+  kind: ActionKind;
+  parameter: string;
+}
+
+export interface ItemInfo {
+  title: string;
+  tooltip?: string;
+  icon: IconDefinition;
+  enabled: boolean;
+  action?: ItemAction;
+}
+
+export const Items: readonly ItemInfo[] = [
+  {
+    title: 'Getting Started',
+    icon: faExternalLinkAlt,
+    enabled: true,
+    action: {
+      kind: ActionKind.LINK,
+      parameter: 'https://podman-desktop.io/docs/intro',
```

Comment:
It cannot be reused right now without additional changes on core side, because packages/* included into electron app assembly without package.json files

![image](https://github.com/user-attachments/assets/512903ef-c5e8-4928-b72c-0c1092c518cc)

if we include package.json into assembly as main/package.json, then it can be referenced at runtime/devtime with relative path.

Reply:
It is actually right there as a sibling of packages folder.

Reply:
It looks like vite build just bundles package.json into cjs file. Tested on macOS. Works for me.